### ISSUE TITLE
Client Allocation Request for: Simba

### ISSUE THREAD

**Issue by xinaxu** [filecoin-project/filecoin-plus-client-onboarding#149]
---

# Client Allocation Questions 

## Core Information
- Name: Simba
- Website / Social Media: Slack Xinan Xu
- Region: North America
- DataCap Requested: 5TiB
- Addresses to be Notarized: f1wrzhpcsq4agpqugkjqiht7smjedgrzcsmi4iddq
- Notary Requested: ann-shin

**Comment by ann-shin** [comment]
Hey, @xinaxu ,thanks for your request. Can you kindly add more information / context here on:

- Are you representing an individual or a company？

- If you are representing a company, Please answer the following questions: What are the establishment time, number of employees, business scope, number of registered users, number of social media followers, and storage usage of your company? e.g. include links to social media/news mentions etc. 
If you are an individual, Please introduce yourself,e.g. include links to social media/news mentions, etc.
- Your history working on IPFS / Filecoin, e.g. notable contributions to the community, type of engagement shown over time, etc.
- Type, size, and source of the data you will be store on Filecoin,e.g. public data or private data, etc. Is there any sensitive content or illegal content?
- How you intend to store the data on Filecoin, e.g. redundancy, location/miner preferences, etc
- Have you already apply for DataCap to another notary, and has it been accepted or rejected.
- Would you be OK in posting deal transactions on Github?

**Comment by xinaxu** [comment]
1. I am an individual
2. I am currently a software engineer. I used to be a researcher specialized in Physics. https://www.linkedin.com/in/xinan-xu-b868a326/
3. I have been an active slack member starting October, I've been developing filecoin related tools in my github repo and actively reporting bugs related to Filecoin/lotus. I am a participant of Slingshot 2.x and I am also a Filecoin storage provider (f064218).
4. I will be storing CCAFS Climate data http://www.ccafs-climate.org/, Data size is 6.9 TiB and most will be stored as compressed ASCII files. The data is licensed under Creative Commons Attribution 4.0 International License which in other words is free to share and redistribute in any format.
5. The data will be stored primarily with 5-10 storage provider located in North America(primarily) and Europe. I will reach out to miners via Slack and verify their identities and geolocations to make sure the data will be distributed evenly and fairly.
6. I have applied for DataCap with Filecoin Foundation. https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/112. The request has been approved. We need slightly more than granted amount to store the complete dataset.
7. Absolutely okay with posting transactions here.
